giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/rest/FilesController.java

```diff
@@ -160,6 +163,44 @@
         return new ResponseEntity<String>(sw.toString(), HttpStatus.OK);
     }
 
+    @TokenCheck
+    @RequestMapping(value = GET_DOCUMENT_PATH
+            + "/access/change", produces = "application/json;charset=UTF-8", method = RequestMethod.POST)
+    public ResponseEntity<String> changeDocumentAccess(@RequestParam(defaultValue = "") String accessToken,
```

Comment:
method should be in it's own file and access parameter should be case-insensitive. (it shouldn't matter if user submits "private" or "PRIVATE".